File: module_common/src/main/java/module/common/utils/LogUtils.java
package module.common.utils;

import android.util.Log;

public class LogUtils {

    /*是否打印日志*/
    private static boolean isOpen = true;

    private static final String TAG = "LogUtils";

    public static void i(String tag,String msg){
        if(isOpen){
            Log.i(tag, msg);
        }
    }

    public static void e(String tag,String msg){
        if(isOpen){
            Log.e(tag, msg);
        }
    }

    public static void i(String msg){
        if(isOpen){
            Log.i(TAG, msg);
        }
    }

    public static void e(String msg){
        if(isOpen){
            Log.e(TAG, msg);
        }
    }


    public static void setIsOpen(boolean isOpen) {
        LogUtils.isOpen = isOpen;
    }

    public static boolean isIsOpen() {
        return isOpen;
    }
}

File: module_common/src/main/java/module/common/utils/LogUtils.kt
package module.common.utils

import android.util.Log

object LogUtils {
    /*是否打印日志*/
    var isIsOpen = true
        private set

    private const val TAG = "LogUtils"

    fun printI(tag: String?, msg: String?) {
        if (isIsOpen) {
            Log.i(tag, msg!!)
        }
    }

    fun printE(tag: String?, msg: String?) {
        if (isIsOpen) {
            Log.e(tag, msg!!)
        }
    }

    fun printI(msg: String?) {
        if (isIsOpen) {
            Log.i(TAG, msg!!)
        }
    }

    fun printE(msg: String?) {
        if (isIsOpen) {
            Log.e(TAG, msg!!)
        }
    }

    fun setIsOpen(isOpen: Boolean) {
        isIsOpen = isOpen
    }
}